src/core/server.js

```diff
@@ -676,6 +676,17 @@ Server.prototype._handleResponse = function(message) {
     const result = message.result;
     const responseEvent = 'response_' + command;
 
+    if (command === 'submit' && result.engine_result === 'tefPAST_SEQ' &&
```

Comment:
So we don't know if the this transaction already got validated because it could be any new transaction at this point right?